packages/browsers/src/browser-data/chrome.ts

```diff
@@ -86,41 +71,39 @@ export function relativeExecutablePath(
       return path.join('chrome-' + folder(platform), 'chrome.exe');
   }
 }
+
+export async function getLastKnownGoodVersionsForChannel(
```

Comment:
The naming seems off the variable states Versions, and we are getting an object. 
How about `getLastKnownGoodReleaseForChannel`?